From b376cb6750409487f78e22e108bd9c3fd84a6f90 Mon Sep 17 00:00:00 2001
From: Juergen Hoetzel <juergen@archlinux.org>
Date: Fri, 1 Dec 2023 19:58:00 +0100
Subject: [PATCH] daemon/btrfs: Use new syntax for balance command

Support for the obsolete short syntax 'btrfs balance /path' was
removed in btrfs-progs 6.6.
---
 daemon/btrfs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/daemon/btrfs.c b/daemon/btrfs.c
index a145578b3..e6ade50d4 100644
--- a/daemon/btrfs.c
+++ b/daemon/btrfs.c
@@ -551,6 +551,7 @@ do_btrfs_filesystem_balance (const char *fs)
 
   ADD_ARG (argv, i, "btrfs");
   ADD_ARG (argv, i, "balance");
+  ADD_ARG (argv, i, "start");
   ADD_ARG (argv, i, fs_buf);
   ADD_ARG (argv, i, NULL);
 
-- 
2.43.0

